src/Pester.RSpec.ps1

```diff
@@ -373,10 +373,10 @@ function New-PesterContainer {
     [CmdletBinding(DefaultParameterSetName="Path")]
     param(
         [Parameter(Mandatory, ParameterSetName = "Path")]
-        [String] $Path,
+        [String[]] $Path,
```

Comment:
Now that this cmdlet generates the containers, it seems like a waste to not allow it to grab an array of files / scriptblocks. This is very useful when you don't pass any data, but might also be used when passing the same set of data to multiple sets of paths.